### DISCUSSION TITLE
Feature Request: [VBA Mode] Project-level setting

### DISCUSSION THREAD

**Discussion by mburns08109** [twinbasic/twinbasic#448] [Ideas]
Per this MS document: https://docs.microsoft.com/en-us/openspecs/microsoft_general_purpose_programming_languages/ms-vbal/a09fd48e-abed-4da8-8c4c-a110bf4ef6b6?redirectedfrom=MSDN

MS VBA sets class instancing as Private as the default.
tB defaults to Public.

Allow a project-level option setting to toggle the default to match VBA.

**Comment by bclothier** [comment]
Can anyone use a DLL that contains only private classes, though? Probably not. An EXE would be able to get away with that, though. Sounds like a potential compiler warning.

I think having a setting to specify the default visibility of new classes would be good -- I'm more likely to want a `Private Class` rather than `Public Class` and I really dislike just `Class`.
